lib/rom/setup/auto_registration.rb

```diff
@@ -44,7 +44,7 @@ def load_entities(entity)
     def const_name(entity, file)
       name =
         if namespace
-          file.gsub("#{directory.dirname}/", '')
+          "#{namespace}/#{entity}/#{File.basename(file, '.rb')}"
```

Comment:
I don't think this works well on windows; have to use `File.join` or something else that chooses the correct path separator